spec/lib/puma/appsignal_spec.rb

```diff
@@ -68,6 +89,13 @@ def create(&block)
     wait_for("enough probe calls") { probe.calls >= 2 }
   end
 
+  it "marks the PumaProbe thread as fork-safe", :not_ruby19 do
+    out_stream = std_stream
+    capture_stdout(out_stream) { Puma.run }
+
+    expect(out_stream.read).not_to match(/WARNING: Detected 1 Thread/)
```

Comment:
```suggestion
    expect(out_stream.read).not_to include("WARNING: Detected 1 Thread")
```

I tested locally that this works. Then we don't need to use a regex :)